docs/plugins/argocd.md

```diff
@@ -10,13 +10,13 @@ This plugin installs [ArgoCD](https://argoproj.github.io/cd/) in an existing Kub
 
 ### Default Configs
 
-| key | default value | description |
-|  ----  | ----  | ----  |
-| chart.chart_name | argo/argo-cd | argocd's official chart name |
-| chart.timeout | 5m | this config will wait 5 minutes to deploy argocd |
-| upgradeCRDs | true | default update CRD config |
-| chart.wait | true | whether to wait until install is complete |
-| repo.url | https://argoproj.github.io/argo-helm | helm repo address |
-| repo.name | argo | helm repo name |
+|       key        | default value                        | description                                      |
+|       ----       | ----                                 | ----                                             |
+| chart.chart_name | argo/argo-cd                         | chart name                                       |
+|  chart.timeout   | 5m                                   | this config will wait 5 minutes to deploy argocd |
+|   upgradeCRDs    | true                                 | default update CRD config                        |
+|    chart.wait    | true                                 | whether to wait until install is complete        |
+|     repo.url     | https://argoproj.github.io/argo-helm | helm official repo address                       |
+|    repo.name     | argo                                 | helm repo name                                   |
```

Comment:
I see that in the first column of your table, some of the documents are center-aligned like this, and some of them are left-aligned.

Reply:
After testing, I found that if you edit the file a bit (e.g. delete a space in the table and add it back), then Ctrl+S, Goland will automatically align it to the left for you. Try it?

Reply:
> it's editor's operation， which one looks better

I prefer the left-aligned one?